config/default/manager/manager.yaml

```diff
@@ -3,9 +3,9 @@ kind: Namespace
 metadata:
   labels:
     control-plane: mac-controller-manager
-    controller-tools.k8s.io: "1.0"
-    runlevel: "0"
-    openshift.io/run-level: "0"
+    # Make sure that KubeMacPool pods are never handled by the webhook to
+    # prevent dead-lock.
+    mutatepods.kubemacpool.io: ignore
```

Comment:
maybe we should also add the label of virtualmachines with ignore value?

Reply:
The point here is to make sure we don't endup in a deadlock where a Pod that implements KMP is not stuck on creation waiting for itself. Unless we add management VMs implementing KMP, we don't need to ignore them.

Reply:
you're right.